yjit/src/cruby.rs

```diff
@@ -124,6 +124,10 @@ extern "C" {
 
     pub fn rb_hash_new_with_size(sz:usize) -> VALUE;
     pub fn rb_hash_bulk_insert(argc:c_long, argv: *mut u8, hash:VALUE);
+    pub fn rb_hash_resurrect(hash:VALUE) -> VALUE;
+
+    pub fn rb_ec_ary_new_from_values(ec: EcPtr, n:c_long, elts: *const VALUE) -> VALUE;
+    pub fn rb_ary_resurrect(ary: VALUE) -> VALUE;
```

Comment:
We should generate these bindings with bindgen

Reply:
Okay. I'm having a really interesting time doing that in the obvious way. I'll keep at it.

Reply:
Can't get the bindgen tool to run? If that's the case I'd recommend posting in #yjit and tagging Alan. I haven't tried to run his bindgen tool yet and would be curious to hear what he says.